apps/web/modules/auth/lib/utils.ts

```
@@ -228,46 +228,47 @@ export const shouldLogAuthFailure = async (
+    if (recentEntries.length === 0) return true;
 
-      return currentCount % 10 === 0 || timeSinceLastLog > 60000;
-    } catch (error) {
-      logger.warn("Redis rate limiting failed, not logging due to Redis requirement", { error });
-      // If Redis fails, do not log as Redis is required for audit logs
-      return false;
-    }
-  } else {
-    logger.warn("Redis not available for rate limiting, not logging due to Redis requirement");
-    // If Redis not configured, do not log as Redis is required for audit logs
+    const lastLogTime = parseInt(recentEntries[recentEntries.length - 1].split(":")[0]);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)